amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/fileplans/FilePlanTests.java

```diff
@@ -594,4 +603,178 @@ public void listHolds()
             }
         });
     }
+
+    /**
+     * <pre>
+     * Given that a file plan exists
+     * When rmAdmin user ask the API for roles and relations
+     * It provides list of all roles and relations
+     * </pre>
+     */
+    @Test
+    public void listFilePlanAllRolesAndRelation()
+    {
+        // Call to new API to get the roles and capabilities
+        Object response = getRestAPIFactory().getFilePlansAPI().getFilePlanRolesAndRelation(FILE_PLAN_ALIAS, "");
```

Comment:
In those test we should use `RoleCollection` instead `Object` and use it in a similar way as in hold tests. Moreover, we should check if a role contains expected capabilities. 
Moreover, the  test only checks the number of roles returned. Can get a list of default roles and check if the endpoint returns that roles?